chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
 
 \codeexample{simple-sample.do}{./code/simple-sample.do}
 
+% what to think about before sampling
 Almost all of the relevant considerations for sampling come from two sources:
 deciding what population, if any, a sample is meant to represent (including subgroups);
-and deciding that different individuals should have different probabilities
+and deciding whether different individuals should have different probabilities
 of being included in the sample.
 These should be determined in advance by the research design,
 since otherwise the sampling process will not be clear,
 and the interpretation of measurements is directly linked to who is included in them.
 Often, data collection can be designed to keep complications to a minimum,
 so long as it is carefully thought through from this perspective.
-Ex post changes to the study scope using a sample drawn for a different purpose
+Ex-post changes to the study scope using a sample drawn for a different purpose
 usually involve tedious calculations of probabilities and should be avoided.
 
 
-\subsection{Randomization}
+\subsection{Randomized treatment assignment}
 
-\textbf{Randomization}, in this context, is the process of assigning units into treatment arms.
-Most of the code processes used for randomization are the same as those used for sampling,
-since randomization is also a process of splitting a sample into groups.
+% what does randomized assignment mean
+\textbf{Randomized treatment assignment} is the process of assigning observations into treatment arms.
+Most of the code processes used for random assignment are the same as those used for sampling,
+since it is also a process of splitting a a list of observations into groups.
 Where sampling determines whether a particular individual
 will be observed at all in the course of data collection,
-randomization determines if each individual will be observed
-as a treatment unit or used as a counterfactual.
-Randomizing a treatment guarantees that, \textit{on average},
-the treatment will not be correlated with anything but the results of that treatment.\cite{duflo2007using}
-Causal inference from randomization therefore depends on a specific counterfactual:
-that the units who received the treatment program
-could just as well have been randomized into the control group.
-Therefore, controlling the exact probability that each individual receives treatment
-is the most important part of a randomization process,
-and must be carefully worked out in more complex designs.
-
-Just like sampling, the simplest form of randomization is a uniform-probability process.\sidenote{
-  \url{https://dimewiki.worldbank.org/Randomization_in_Stata}}
+random assignment determines if each individual will be observed
+as a treatment observation or used as a counterfactual.
+Randomizing a treatment assignment guarantees that, \textit{on average},
+the treatment will not be correlated with anything but the impact of that treatment.\cite{duflo2007using}
```

Comment:
this should go in experimental design section above, and then this section should be clearly linked to experimental design

Reply:
done in 069a6453a0ac43e44191eeca4071fa2e106ad6c1